app/database/db_build.sql

```diff
@@ -0,0 +1,16 @@
+BEGIN;
+
+DROP TABLE IF EXISTS nuggets;
+
+CREATE TABLE nuggets(
+  id SERIAL PRIMARY KEY,
+  x_coordinate INTEGER NOT NULL,
+  y_coordinate INTEGER NOT NULL,
```

Comment:
http://stackoverflow.com/questions/1196415/what-datatype-to-use-when-storing-latitude-and-longitude-data-in-sql-databases